SKAToolKit/Categories/NSMutableDictionary+SKATMXParserUtils.m

```diff
@@ -54,151 +54,119 @@
 @implementation NSMutableDictionary (SKATMXParserUtils)
 
 #pragma mark - Public
-- (void)cleanseMap
-{
-    [self cleanseProperties];
+- (void)cleanseMap {
```

Comment:
Our coding standard for this project is to put opening braces on new lines. All of these opening braces should stay how they were.